src/solutions/how-tos/confluent-cloud/iot-ingest-control.md

```diff
@@ -0,0 +1,455 @@
+      "api_secret":"<cc-api-secret>",
+      "bootstrap_server":"<cc-bootstrap-server>",
+    }
+    ```
+
+- Secret Name: `zilla-cc-access`
+- Review and store the secret
+
+### Create the <ZillaPlus/> proxy VPC
+
+> This creates your <ZillaPlus/> proxy VPC in preparation for access via the internet.
+
+[Create a VPC plus other VPC resources](https://docs.aws.amazon.com/vpc/latest/userguide/create-vpc.html#create-vpc-and-other-resources) with the below resource names.
+
+- Name tag auto-generation: `my-cc-iot`
+- VPC endpoints: `none`
+- Create the VPC
+
+### Create the <ZillaPlus/> proxy security group
+
+> This creates your <ZillaPlus/> proxy security group to allow Kafka clients and SSH access.
+
+A VPC security group is needed for the <ZillaPlus/> proxies when they are launched.
+
+Follow the [Create Security Group](https://console.aws.amazon.com/vpcconsole/home#CreateSecurityGroup:) wizard with the following parameters and defaults. This creates your <ZillaPlus/> proxy security group to allow Kafka clients and SSH access.
+
+::: note Check your selected region
+Make sure you have selected the desired region, such as `US East (N. Virginia) us-east-1`.
+:::
+
+- Name: `my-zilla-iot-proxy-sg`
+- VPC: `my-cc-iot-vpc`
+- Description: `MQTT clients and SSH access`
+- Add Inbound Rule
+  - Type: `CUSTOM TCP`
+  - Port Range: `8883`
+  - Source type: `Anywhere-IPv4`
+- Add Inbound Rule
+  - Type: `SSH`
+  - Source type: `My IP`
+- Add Outbound Rule (if not exists)
+  - Type: `All traffic`
+  - Destination: `Anywhere-IPv4`
+- Create the Security Group
+
+::: warning Check your network settings
+Your IP may be different when you SSH into the EC2 instance. VPNs and other networking infrastructure may cause the `My IP` inbound rule to fail. Instead, you can use one of the other ways AWS provides to execute commands in an EC2 instance.
+:::
+
+Navigate to the VPC Management Console [Security Groups](https://console.aws.amazon.com/vpc/home#securityGroups:) table. Select the `my-zilla-iot-proxy-sg` security group you just created. You will create an inbound rule to allow all traffic inside itself.
+
+- Add Inbound Rule
+  - Type: `All Traffic`
+  - Source type: `Custom`
+  - Source: `my-zilla-iot-proxy-sg`
+
+Add the `my-zilla-iot-proxy-sg` security group to your VPC Endpoint by finding your `my-cc-iot-vpce` from the [Endpoints table](https://console.aws.amazon.com/vpcconsole/home#Endpoints:).
+
+- Select your VPC endpoint
+- `Actions` menu > select `Manage Security Groups`
+- Select both security groups:
+  - `default`
+  - `my-zilla-iot-proxy-sg`
+- Save the changes
+
+### Create the <ZillaPlus/> proxy IAM security role
+
+> This creates an IAM security role to enable the required AWS services for the <ZillaPlus/> proxies.
+
+Follow the [Create IAM Role](./../aws-services/create-iam-role.md) guide to create an IAM security role with the following parameters:
+
+::: code-tabs
+
+@tab Name
+
+```text:no-line-numbers
+aklivity-zilla-proxy
```

Comment:
Seems like this naming is inconsistent with the rest, suggest we unify the naming pattern, so that permissions can be set differently for different secrets used by different deployments more easily.

```suggestion
my-zilla-iot-role
```